### DISCUSSION TITLE
Shortcut to show Type Information of element under cursor

### DISCUSSION THREAD

**Discussion by ghost** [zed-industries/zed#7477] [Feature Requests]
I saw a similar question #6489, but this is not exactly the same. Sometimes you don't need the complete documentation in the tooltip, just the Type Information. I think this is important in languages like Rust / Haskell, etc. 

Thanks in advance.

**Comment by mrnugget** [comment]
The problem is that you don't want the documentation in here? 
![screenshot-2024-02-07-09 26 33@2x](https://github.com/zed-industries/zed/assets/1185253/b987e348-79d0-411b-9a75-ae3a8139d9a4)

Because you can trigger this with `editor: hover` (I have it bound to `shift-k`in Vim)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
